### PR TITLE
add task_id to node logs

### PR DESCRIPTION
**Pull request by isahers1**

### CODE REVIEW

File: libs/langgraph/langgraph/pregel/debug.py [hunk 1]
```diff
@@ -90,9 +90,10 @@ def map_debug_task_results(
     step: int,
     tasks: list[PregelExecutableTask],
     stream_channels_list: Sequence[str],
+    step_start_time: datetime,
 ) -> Iterator[DebugOutputTaskResult]:
-    ts = datetime.now(timezone.utc).isoformat()
-    for name, _, _, writes, config, _, _, _ in tasks:
+    ts = datetime.now(timezone.utc)
+    for name, _, _, writes, config, _, _, task_id in tasks:
```
Comment (vbarda) [inline libs/langgraph/langgraph/pregel/debug.py:96]: should we just use attributes directly since it's a namedtuple? cc @nfcampos

File: libs/langgraph/langgraph/pregel/loop.py [hunk 2]
```diff
@@ -241,7 +241,9 @@ def _output_writes(
                 self.stream.extend(
                     (self.config["configurable"].get("checkpoint_ns", ""), "debug", v)
                     for v in map_debug_task_results(
-                        self.step, [(task, writes)], self.stream_keys
+                        self.step,
+                        [(task, writes)],
+                        self.stream_keys,
```
Comment (nfcampos) [inline libs/langgraph/langgraph/pregel/loop.py:246]: undo this change

**Review by nfcampos — APPROVED** [review]
